build_tools/doc/doc_requirements.txt

```diff
@@ -1,6 +1,6 @@
 Cython>=0.29,<0.29.18
 numpy>=1.16
-scipy>=1.3
```

Comment:
Did here just because technically the docker image we use has not been implemented yet (added in #347)